components/lib/tree/UITreeNode.js

```diff
@@ -961,7 +969,6 @@ export const UITreeNode = React.memo((props) => {
                 'aria-setsize': props.node.children ? props.node.children.length : 0,
                 'aria-posinset': props.index + 1,
                 onKeyDown: onKeyDown,
-                'aria-expanded': expanded,
```

Comment:
Are you sure you need to remove this aria-expanded?

Reply:
it was a duplicated entry so i removed it too due to an eslint error